modules/lib/lib-admin/src/main/resources/lib/xp/admin.ts

```diff
@@ -160,3 +160,19 @@ export function getInstallation(): string {
 export function getVersion(): string {
     return helper.getVersion();
 }
+
+const adminLibrary = {
+    getAssetsUri,
+    getBaseUri,
+    getHomeToolUrl,
+    getInstallation,
+    getLauncherPath,
+    getLauncherUrl,
+    getLocale,
+    getLocales,
+    getPhrases,
+    getToolUrl,
+    getVersion,
+};
+
+export type AdminLibrary = typeof adminLibrary;
```

Comment:
Is `AdminLibrary` not just the same as `XpLibraries["/lib/xp/admin"]`;

We cloud do this: 
```suggestion
export type AdminLibrary = XpLibraries["/lib/xp/admin"];
```

Or this
```suggestion
export type AdminLibrary = typeof import('./admin');
```

Or just use `XpLibraries["/lib/xp/admin"]` directly where we want to use that type.

Reply:
<img width="817" alt="Screenshot 2022-11-23 at 14 34 09" src="https://user-images.githubusercontent.com/118847/203559790-872704d2-8838-4929-b597-cbfeb8489d0c.png">

<img width="1369" alt="Screenshot 2022-11-23 at 14 34 41" src="https://user-images.githubusercontent.com/118847/203559901-493a3d13-e038-4c12-a938-f8344fb5c0e9.png">

Reply:
<img width="1374" alt="Screenshot 2022-11-23 at 14 36 44" src="https://user-images.githubusercontent.com/118847/203560380-e245e073-c02b-4ba1-8b00-3b1039dab3de.png">

Reply:
I think the problem might be that import() with parentheses, is dynamic, while types should be static?

Reply:
Going the opposite way makes hovering even a import file path look nice.

<img width="832" alt="Screenshot 2022-11-23 at 14 40 50" src="https://user-images.githubusercontent.com/118847/203561196-e2152bb3-ea9a-4464-9c2e-da1d83bec13f.png">

Reply:
Dynamic imports of types makes no sense. Dynamic imports are runtime, types are NOT runtime.

Reply:
I still can't figure out the original problem. What exactly are those changes suppose to fix? Do you mean that the type details in the editor are incorrect? Or am I missing something? Can you, please, provide an example of how you use this?

Reply:
So I would do this in all the libs:
<img width="537" alt="Screenshot 2022-11-23 at 15 03 40" src="https://user-images.githubusercontent.com/118847/203566304-1497a3d9-e3e6-4d81-a3aa-4d691c362ef4.png">

Reply:
This was originally about adding a feature not fixing anything.

But along the way I've come across the dynamic import issue.

And just now I found another interesting thing.
When switching from named exports to a default export object, I discovered we are exporting two extra undocumented functions.

Namely:
* modify
* getBinary

If we change from dynamic import to using the default export, then then I don't need any extras. 

<img width="1294" alt="Screenshot 2022-11-23 at 15 29 21" src="https://user-images.githubusercontent.com/118847/203572047-6176c638-2c5f-41d0-92ed-18311d8fe894.png">

Reply:
This Pull Request should not hinder the release.

And the default export was just another experiment, I don't like it.

I need to be able to explain the original "problem", before a potential solution can be implemented.

However I still think the dynamic import is wrong, but could be fixed in another PR?